app/controllers/admin/admin_application_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::AdminApplicationController < ApplicationController
```

Comment:
```suggestion
class Admin::ApplicationController < ApplicationController
```
naming lại được không em ? mấy controller khác nữa, anh thấy dư chữ admin á

Reply:
Dạ được a, để e fix lại